src/ios-deploy/ios-deploy.m

```diff
@@ -1714,38 +1724,72 @@ void handle_device(AMDeviceRef device) {
         check_error(AMDeviceStopSession(device));
         check_error(AMDeviceDisconnect(device));
 
-        CFStringRef keys[] = { CFSTR("PackageType") };
-        CFStringRef values[] = { CFSTR("Developer") };
-        CFDictionaryRef options = CFDictionaryCreate(NULL, (const void **)&keys, (const void **)&values, 1, &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks);
-
-        //assert(AMDeviceTransferApplication(afcFd, path, NULL, transfer_callback, NULL) == 0);
-        check_error(AMDeviceSecureTransferPath(0, device, url, options, transfer_callback, 0));
-        close(*afcFd);
-
-
+        CFDictionaryRef options;
+        if (!incremental_install) {
```

Comment:
Looks like we could remove 'incremental_install' and change it's checks to `app_deltas != NULL`